content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -0,0 +1,577 @@
+
+## Install the Datadog Agent with OpenTelemetry Collector
+
+### Add the Datadog Helm Repository
+
+To add the Datadog repository to your Helm repositories:
+
+```shell
+helm repo add datadog https://helm.datadoghq.com
+helm repo update
+```
+
+### Set up Datadog API and application keys
+
+1. Get the Datadog [API][2] and [application keys][3].
+1. Store the keys as a Kubernetes secret:
+   ```shell
+   kubectl create secret generic datadog-secret \
+     --from-literal api-key=<DD_API_KEY> \
+     --from-literal app-key=<DD_APP_KEY>
+   ```
+   Replace `<DD_API_KEY>` and `<DD_APP_KEY>` with your actual Datadog API and application keys.
+
+### Configure the Datadog Agent
+
+Use a YAML file to specify the Helm chart parameters for the [Datadog Agent chart][4].
+
+1. Create an empty `datadog-values.yaml` file:
+   ```shell
+   touch datadog-values.yaml
+   ```
+   <div class="alert alert-info">Unspecified parameters will use defaults from <a href="https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml">values.yaml</a>.</div>
+1. Configure the Datadog API and Application key secrets:
+   {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
+datadog:
+  apiKeyExistingSecret: datadog-secret
+  appKeyExistingSecret: datadog-secret
+   {{< /code-block >}}
+1. Switch the Datadog Agent Docker repository to use development builds:
+   {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
+agents:
+  image:
+    repository: datadog/agent-dev
+    tag: nightly-ot-beta-main-jmx
+    doNotCheckTag: true
+...
+   {{< /code-block >}}
+   <div class="alert alert-info">This guide uses a Java application example. The <code>-jmx</code> suffix in the image tag enables JMX utilities. For non-Java applications, use <code>nightly-ot-beta-main</code> instead.<br> For more details, see <a href="/containers/guide/autodiscovery-with-jmx/?tab=helm">Autodiscovery and JMX integration guide</a>.</div>
+1. Enable the OpenTelemetry Collector and configure the essential ports:
+   {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
+datadog:
+  ...
+  otelCollector:
+    enabled: true
+    ports:
+      - containerPort: "4317" # default port for OpenTelemetry gRPC receiver.
+        hostPort: "4317"
+        name: otel-grpc
+      - containerPort: "4318" # default port for OpenTelemetry HTTP receiver
+        hostPort: "4318"
+        name: otel-http
+   {{< /code-block >}}
+   <div class="alert alert-info"><p>It is required to set the <code>hostPort</code> in order for the container port to be exposed to the external network. This enables configuring the OTLP exporter to point to the IP address of the node to which the Datadog Agent is assigned.</p><p>Alternatively, if you doesn't want to expose the port, you can use the agent service instead via <code>&lt;SERVICE_NAME&gt;.&lt;SERVICE_NAMESPACE&gt;.svc.cluster.local</code>.</p></div>
```

Comment:
It's not clear to me exactly what to do if you want to use the agent service instead of exposing the port. I'm assuming there's an `<SERVICE_NAME>.<SERVICE_NAMESPACE>.svc.cluster.local` section of the helm chart—what value do you configure for it? is there an example we can point to?

Reply:
Good point! Updated to include instructions on how to change this.